pkg/integrations/windows_exporter/windows_exporter.go

```diff
@@ -0,0 +1,91 @@
+// +build windows
+
+package windows_exporter //nolint:golint
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/gorilla/mux"
+	"github.com/grafana/agent/pkg/integrations/config"
+	"github.com/prometheus-community/windows_exporter/exporter"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+type Integration struct {
+	c      *Config
+	logger log.Logger
+	wc     *exporter.WindowsCollector
+
+	exporterMetricsRegistry *prometheus.Registry
+}
+
+// New creates a new node_exporter integration.
+func New(log log.Logger, c *Config) (*Integration, error) {
+
+	bytes, _ := yaml.Marshal(c.WindowsConfig)
+	cb := string(bytes)
+	wc, _ := exporter.NewWindowsCollector(c.Name(), c.EnabledCollectors, cb)
+
+	level.Info(log).Log("msg", "Enabled windows_exporter collectors")
+
+	return &Integration{
```

Comment:
It looks like you can use NewCollectorIntegration here since there's nothing specific you need to do for the handler.